playbooks/roles/csv/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Create csv dir
+  file:
+    path: /home/vagrant/csv
+    state: directory
+    owner: vagrant
+
+- name: Create operating-systems.csv
+  template:
+    src: templates/operating-systems.csv.j2
+    dest: /home/vagrant/csv/operating-systems.csv
+    owner: vagrant
+
+- name: Run hammer csv operating-systems
+  shell: >
+    BUNDLE_GEMFILE=~/hammer-cli-katello/Gemfile bundle exec {{ csv_hammer }}
+    csv operating-systems
+    --verbose
+    --file /home/vagrant/csv/operating-systems.csv
+    > /home/vagrant/csv/operating-systems.log
+
```

Comment:
Is there some way to put the above two sections into a loop that iterates over all the resource types? Would be better than cut&paste of this section.